src/pages/onboardingModal/final/item.tsx

```diff
@@ -0,0 +1,16 @@
+import { Title } from '@patternfly/react-core';
+import React from 'react';
+
+interface Props {
+  title: string;
+  value: string;
+}
+
+const Item: React.SFC<Props> = ({ title, value }) => (
+  <div style={{ paddingBottom: '12px' }}>
+    <Title size="md">{title}</Title>
+    <Title size="lg">{value}</Title>
```

Comment:
FYI, you can use TitleSize to define the size of this PF4 component.

Reply:
Thanks I will update to use it.